content/docs/examples/tutorial/prereq/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Prerequisites
+overview: Check the prerequisites for this tutorial.
+weight: 2
+---
+
+{{< boilerplate work-in-progress >}}
+
+For this tutorial you need a Kubernetes cluster with a namespace for the tutorial's modules and a
+local computer to run the commands. If you are participating in a guided workshop
+and the instructors provide the cluster, let them handle the Kubernetes cluster prerequisites, and you can skip ahead to
+setup your local computer.
+If you want to proceed using your own cluster, ensure your cluster satisfies the prerequisites for the cluster as well.
+
+## Kubernetes cluster
+
+Ensure the following conditions are met:
+
+- You can access a Kubernetes cluster and have administrator privileges for said cluster.
```

Comment:
Not sure `said` is appropriate. Maybe `You have administrator privileges to a Kubernetes cluster.`

Reply:
@rcaballeromx uses `said`. @rcaballeromx could you please comment?